pkg/events/events.go

```diff
@@ -88,7 +92,7 @@ func (h *HTTPRequest) FillFieldsFromRequest(r *http.Request) {
 	h.URI = r.RequestURI
 	h.Referer = r.Referer()
 	h.RequestID = r.Header.Get("X-Request-ID")
-	h.BytesRead, _ = strconv.Atoi(r.Header.Get("Content-Length")) //nolint: errcheck
+	h.BytesRead, _ = strconv.Atoi(r.Header.Get("Content-Length")) //nolint: errcheck // Why: best effort
```

Comment:
This will override `BytesRead` with a `0` on error. Is that desired? It looks kind of like a bug (or at least an unintended side-effect).

This whole line looks fairly sketchy tbh. The amount of bytes a request declares itself to be sending in no way should affect the number of bytes read while sending a request, which is what the target field seems to be.

Reply:
So this is creating a custom type for logging. We're not really overriding anything here. We are best effort initializing to the information that we get from the request.